Jellyfin/Views/VideoViewModel.cs

```diff
@@ -70,12 +81,134 @@ public async void HandleParameters(Video.Parameters parameters)
+                            MediaSourceId = videoId.ToString("N"),
+                            AudioStreamIndex = parameters.AudioStreamIndex,
+                            SubtitleStreamIndex = parameters.SubtitleStreamIndex,
+                            PlaySessionId = _playingSessionId,
+                            PositionTicks = await GetCurrentTicks(),
+                            SessionId = _appSettings.SessionId,
+                            IsPaused = true
+                        });
+                        break;
+                    case MediaPlaybackState.Playing:
+                        _ = _jellyfinApiClient.Sessions.Playing.Progress.PostAsync(new PlaybackProgressInfo()
+                        {
+                            CanSeek = true,
+                            ItemId = videoId,
+                            MediaSourceId = videoId.ToString("N"),
+                            AudioStreamIndex = parameters.AudioStreamIndex,
+                            SubtitleStreamIndex = parameters.SubtitleStreamIndex,
+                            PlaySessionId = _playingSessionId,
+                            PositionTicks = await GetCurrentTicks(),
+                            SessionId = _appSettings.SessionId,
+                            IsPaused = false
+                        });
+                        break;
+                }
+            };
+
+            await _jellyfinApiClient.PlayingItems[videoId].PostAsync(request =>
+            {
+                request.QueryParameters.MediaSourceId = videoId.ToString("N");
+                request.QueryParameters.AudioStreamIndex = parameters.AudioStreamIndex;
+                request.QueryParameters.SubtitleStreamIndex = parameters.SubtitleStreamIndex;
+                request.QueryParameters.PlaySessionId = _playingSessionId = playbackInfo.PlaySessionId;
+                // TODO: do we need to support sessions/sessionId?
+                request.QueryParameters.CanSeek = _playerElement.MediaPlayer.PlaybackSession.CanSeek;
+
+            });
+
+            _progressTimer = new Timer(async (o) =>
+            {
+                if (await IsPlaying())
+                {
+                    Int64 currentTicks = await GetCurrentTicks();
+                    _ = _jellyfinApiClient.PlayingItems[videoId].Progress.PostAsync(request =>
+                    {
+                        request.QueryParameters.MediaSourceId = videoId.ToString("N");
+                        request.QueryParameters.AudioStreamIndex = parameters.AudioStreamIndex;
+                        request.QueryParameters.SubtitleStreamIndex = parameters.SubtitleStreamIndex;
+                        request.QueryParameters.PlaySessionId = _playingSessionId = playbackInfo.PlaySessionId;
+                        request.QueryParameters.PositionTicks = currentTicks;
+
+                    });
+                    _ = _jellyfinApiClient.Sessions.Playing.Progress.PostAsync(new PlaybackProgressInfo()
```

Comment:
Do you need this *and* the call above? I'm not sure I understand the difference.